XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs

```diff
@@ -316,7 +319,6 @@ private void credentials_TextChanged(object sender, EventArgs e)
 
         private void xsCredentials_TextChanged(object sender, EventArgs e)
         {
-            UpdateButtons();

```

Comment:
By removing this line, the OK button stays disabled even after the XS credentials have been entered (but not checked with the "Test Ctedentials" button) 
Was this change intentional?